docs/apis/cli.md

```diff
@@ -185,6 +189,18 @@ Action "list" lists running and scheduled programs.
      -s,--scheduled                Show only scheduled programs and their JobIDs
 
 
+Action "stop" stops a running program (streaming jobs only).
+
+  Syntax: stop [OPTIONS] <Job ID>
+  "stop" action options:
+     -m,--jobmanager <host:port>   Address of the JobManager (master) to which
+                                   to connect. Specify 'yarn-cluster' as the
+                                   JobManager to deploy a YARN cluster for the
+                                   job. Use this flag to connect to a different
```

Comment:
I never tried it in a yarn cluster. Do you think it needs special handling? Basically, the "stop" signal is sent to the JobManager which forward the signal to all sources. This should be independent if yarn is involved or not. But I don't have any experience with yarn. Tell me if I am wrong.